core/optaplanner-core-impl/src/main/java/org/optaplanner/core/impl/heuristic/selector/entity/EntitySelector.java

```diff
@@ -8,6 +8,9 @@
 
 /**
  * Selects instances of 1 {@link PlanningEntity} annotated class.
+ * It is expected that if two instances share the same properties,
+ * they will be {@link Object#equals(Object) equal} to one another.
```

Comment:
(soft) style: use present tense https://developers.google.com/style/tense
```suggestion
 * they are {@link Object#equals(Object) equal} to one another.
```